src/tests/GfxReplayTest.cpp

```diff
@@ -344,13 +348,15 @@ void GfxReplayTest::testPlayer() {
 }
 
 void GfxReplayTest::testPlayerReadMissingFile() {
-  auto dummyAssetsCallback =
+  auto callbacks = esp::gfx::replay::createSceneGraphPlayerCallbacks();
+  callbacks.loadAndCreateRenderInstance_ =
       [&](const esp::assets::AssetInfo& assetInfo,
           const esp::assets::RenderAssetInstanceCreationInfo& creation) {
         return nullptr;
       };
-  auto dummyLightsCallback = [&](const esp::gfx::LightSetup& lights) -> void {};
-  esp::gfx::replay::Player player(dummyAssetsCallback, dummyLightsCallback);
+  callbacks.changeLightSetup_ =
+      [&](const esp::gfx::LightSetup& lights) -> void {};
```

Comment:
```suggestion
      [](const esp::gfx::LightSetup& lights) -> void {};
```

Maybe just my personal issue, but I don't like using capturing lambdas by default. The `std::function` overhead is bad enough as is, no need to make it even worse :) Same elsewhere.